core/domain/stats_services.py

```diff
@@ -251,6 +251,66 @@ def get_stats_for_new_exp_version(
     return exploration_stats
 
 
+def _get_corresponding_exp_issue(
```

Comment:
Nit: suggest moving this under the new public API method below (to treat it more as an 'appendix' helper in terms of how the service is organized).